recipes/pygeoboundaries/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pygeoboundaries" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pygeoboundaries-{{ version }}.tar.gz
+  sha256: 1ac9b1742ccfce974057697cca907cac705eb559b1a6df0647b077989f58b216
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9,<4.0
+    - poetry-core
+    - pip
+  run:
+    - python >=3.9.0,<4.0.0
+    - requests
+    - pandas
+    - geopandas  
+
+test:
+  imports:
+    - pygeoboundaries
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/wmgeolab/pyGeoBoundaries
+  summary: For performing data checks and file validation on geoboundary files. The primary purpose of this package is to facilitate the review process for pull requests made to the geoBoundaries repository.
+  license: CC-BY-4.0
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - DanRunfola
```

Comment:
@rohith4444 it is odd that the submitter is not included here. While we do have DanRunfola OK already, will you list yourself here?